src/Model.ts

```diff
@@ -239,7 +228,7 @@ export default class Model extends Base {
   @action
   save (
     attributes?: {},
-    { optimistic = true, patch = false, keepChanges = true, ...otherOptions }: SaveOptions = {}
+    { optimistic = true, patch = false, keepChanges = false, ...otherOptions }: SaveOptions = {}
```

Comment:
what is this `keepChanges` flag?

Reply:
This was introduced in the 3.0.0 alpha PR. From description:
> Saving keeps changes that were made after the request started but before it finished (can be disabled by passing the option keepChanges = false).

And you reverted the change in [this commit](https://github.com/masylum/mobx-rest/pull/56/commits/2410794130f8581027104cfce21b4fd16c74e79e)